AmahiAnywhere/AmahiAnywhere/Base/BaseUITableViewController.swift

```diff
@@ -10,4 +10,10 @@ import UIKit
 
 class BaseUITableViewController: UITableViewController {
 
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        // Remove Back Button Text
+        navigationItem.title = ""
```

Comment:
Good catch. I just verified this. It could be great if

1) the top one has `Amahi` and
2) the top server level area has the name of the server at the top o the nav screen (instead of above the list of shares -- perhaps making that list of shares look like a folder. In that case, perhaps having "Shares for %s" with %s being the name of the server would work better.

Reply:
Nice @csoni111 We will have to add each view controller's title with code. I will make this update